content/en/guides/features/rendering-modes.md

```diff
@@ -25,9 +25,9 @@ You do not need to add `ssr: true` to your nuxt config in order to enable server
 
 </base-alert>
 
-## SPA
+## Client Side Rendering Only
```

Comment:
:(

Reply:
https://github.com/nuxt/nuxtjs.org/pull/844/files/4b5d3b0255162e42766b658ff4a5cefbd939f8d0#diff-1157ae4b8a56bff80f2c5177b0dfa526 this section is not related to commands. It is not about wording